pkg/action/show.go

```diff
@@ -87,9 +89,17 @@ func (s *Show) Run(chartpath string) (string, error) {
 		if s.OutputFormat == ShowAll {
 			fmt.Fprintln(&out, "---")
 		}
-		for _, f := range s.chart.Raw {
-			if f.Name == chartutil.ValuesfileName {
-				fmt.Fprintln(&out, string(f.Data))
+		if s.JSONPathTemplate != "" {
+			printer, err := printers.NewJSONPathPrinter(s.JSONPathTemplate)
+			if err != nil {
+				return "", fmt.Errorf("error parsing jsonpath %s, %v", s.JSONPathTemplate, err)
```

Comment:
Better to use `errors.Wrapf`. Pattern used throughout codebase including https://github.com/helm/helm/blob/master/pkg/action/install.go#L144.

Reply:
Thank you for your detailed instructions！ Done;